shell/app-shell/elements/arc-planner.js

```diff
@@ -62,10 +62,11 @@ class ArcPlanner extends Xen.Debug(Xen.Base, log) {
     }
   }
   async _createPlanificator(arc, userid) {
-    const protocol = ArcsUtils.getUrlParam('planificatorProtocol') || 'volatile';
+    const storageKeyBase = ArcsUtils.getUrlParam('storageKeyBase') || 'volatile://';
```

Comment:
I'm leaning the other way on this because I would like to see us adopt a single config for storageKeyBase across the code-base.  Can we discuss f2f?

Reply:
okay will allow for plannerStorageKeyBase